catalog/RUB/RUB_RUBCLIM_LCZ_global_lcz_map_v1.jsonnet

```diff
@@ -0,0 +1,162 @@
+local id = 'RUB/RUBCLIM/LCZ/global_lcz_map/v1';
+local subdir = 'RUB';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.proprietary;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'Global map of Local Climate Zones',
+  version: 'v1',
+  'gee:type': ee_const.gee_type.image_collection,
+  description: |||
+    Since their introduction in 2012, Local Climate Zones (LCZs) emerged as a new standard
+    for characterizing urban landscapes, providing a holistic classification approach that
+    takes into account micro-scale land-cover and associated physical properties.
+
+    This global map of Local Climate Zones, at 100m resolution and representative for the
```

Comment:
with 100m pixel size

("resolution" is often used to mean "pixel size", but they are not quite the same: https://www.nrcan.gc.ca/maps-tools-and-publications/satellite-imagery-and-air-photos/tutorial-fundamentals-remote-sensing/satellites-and-sensors/spatial-resolution-pixel-size-and-scale/9407)